automatic/ruby.portable/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,13 @@
+﻿$ErrorActionPreference = 'Stop'
+$toolsDir              = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+
+$packageArgs = @{
+  FileFullPath   = Join-Path $toolsDir 'rubyinstaller-3.1.0-1-x86_x32.7z'
+  FileFullPath64 = Join-Path $toolsDir 'rubyinstaller-3.1.0-1-x64_x64.7z'
+  Destination    = Join-Path $toolsDir 'ruby'
+  PackageName    = 'ruby.portable'
```

Comment:
Instead of changing the package name to a fixed value, can you please change it to

```suggestion
  packageName = $env:ChocolateyPackageName
```